src/django_components/slots.py

```diff
@@ -80,27 +80,26 @@ class SlotFill(NamedTuple):
     is_filled: bool
     nodelist: NodeList
     context_data: Dict
-    alias: Optional[AliasName]
-    scope: Optional[ScopeName]
+    slot_default_var: Optional[SlotDefaultName]
+    slot_data_var: Optional[SlotDataName]
 
 
-class UserSlotVar:
+class LazySlot:
```

Comment:
Renamed and changed so that the bound slot + context is rendered as `{{ slot }}` instead of `{{ slot.default }}`.